src/main/app/models/yesNoOption.ts

```diff
@@ -13,7 +13,7 @@ export class YesNoOption {
     } else if (input === 'no') {
```

Comment:
why don't we extract "yes" and "no" to const and reuse it. Both are used twice in this file.